packages/manager/apps/dedicated/client/app/license/license.controller.js

```diff
@@ -141,6 +141,10 @@ export default /* @ngInject */ (
 
   $scope.resetAction();
 
+  $scope.letterCase = (content) => {
+    return content[0].toUpperCase() + content.slice(1).toLowerCase();
```

Comment:
```suggestion
    return content[0].toUpperCase() + content.slice(1).toLowerCase();
```
As you already use lodash here, you can remove this function and use ```lodash.capitalize```

Reply:
@darsene done